### ISSUE TITLE
What is this, and how does it work?

### ISSUE THREAD

**Issue by Mazedotexe** [AshtonDavies/bookmarklet-executor#1]
I can run javascript on that page, obviously, but it seems like its purpose is to run bookmarklets on any page, even if the bookmarklet is blocked. If this is correct, how does it work?
Thanks in advance

**Comment by AshtonDavies** [comment]
It's just the landing page for the DNS. This is a new repository, so there's no readme with details yet.

**Comment by hankypoo7** [comment]
> It's just the landing page for the DNS. This is a new repository, so there's no README with details yet. It also does not currently have side panel execution to run bookmarklets anywhere.
> 
> DNS details are on [Chrose](https://chrose.netlify.app/detail/Bookmarklet%20Executor).

I know I am extremely late but have you been working on this?

**Comment by AshtonDavies** [comment]
> I know I am extremely late but have you been working on this?

Nah, it's been cancelled. [Reference comment](https://github.com/3kh0/ext-remover/discussions/845#discussioncomment-9514732)
